lib/ansible/modules/monitoring/zabbix/zabbix_screen.py

```diff
@@ -31,15 +31,55 @@
     screens:
         description:
             - List of screens to be created/updated/deleted (see example).
-            - If a screen has already been added, the screen name won't be updated.
-            - When creating or updating a screen, C(screen_name) and C(host_group) are required.
-            - When deleting a screen, the C(screen_name) is required.
-            - Option C(graphs_in_row) will limit columns of a screen and make multiple rows (default 3).
-            - >
-              The available states are: C(present) (default) and C(absent). If the screen already exists, and the state is not C(absent), the screen
-              will be updated as needed.
-            - To sort hosts alphabetically, set the C(sort) option to C(true)
-        required: true
```

Comment:
I think `required: true` should the kept, shouldn't it?

Reply:
Yes, definitely! I guess it got removed accidentally...

Reply:
It did. I've fixed it.